Services/Models/Proto/GPSPacket.cs

```diff
@@ -0,0 +1,340 @@
+// <auto-generated>
+//     Generated by the protocol buffer compiler.  DO NOT EDIT!
+//     source: GPSPacket.proto
```

Comment:
Include in the PR also the original .proto file, so users can use it as a starting point for future extensions, and recreate this file too in case of major protobuf changes.

For security reasons, before accepting the PR we'll also have to compare this code with the output generated by protobuf compiler.